compiler/ballerina-backend-jvm-old/src/main/ballerina/src/compiler_backend_jvm/jvm_type_gen.bal

```diff
@@ -1101,11 +1101,19 @@ function loadInvokableType(jvm:MethodVisitor mv, bir:BInvokableType bType) {
         i += 1;
     }
 
+    bir:BType? restType = bType.restType;
+    if (restType is ()) {
```

Comment:
AFAIR, loadtype() method handles () types also, you don't need this check.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/19976/commits/a14913890adeae9cc25ddc3edb9ecdb7632cf170

Also removed same check on other areas (record rest param check)